src/qt/intro.cpp

```diff
@@ -238,18 +238,6 @@ bool Intro::showIfNeeded(bool& did_show_intro, int64_t& prune_MiB)
                 /* Cancel clicked */
                 return false;
             }
-            dataDir = intro.getDataDirectory();
-            try {
-                if (TryCreateDirectories(GUIUtil::QStringToPath(dataDir))) {
-                    // If a new data directory has been created, make wallets subdirectory too
-                    TryCreateDirectories(GUIUtil::QStringToPath(dataDir) / "wallets");
-                }
-                break;
-            } catch (const fs::filesystem_error&) {
-                QMessageBox::critical(nullptr, PACKAGE_NAME,
-                    tr("Error: Specified data directory \"%1\" cannot be created.").arg(dataDir));
-                /* fall through, back to choosing screen */
-            }
```

Comment:
In commit "Fix wallet directory initialization" (17b4955e4b1e4f02f00a948b432f18caa9b250a5)

Is it a mistake to delete this whole block of code? I'd expect only line 245 (`TryCreateDirectories("wallets")`) to be deleted

Reply:
fixed